Content.Server/Armor/ArmorComponent.cs

```diff
@@ -7,5 +7,13 @@ public sealed class ArmorComponent : Component
     {
         [DataField("modifiers", required: true)]
         public DamageModifierSet Modifiers = default!;
+
+        /// <summary>
+        ///     The examine group used for grouping together examine details.
+        /// </summary>
+        [DataField("examineGroup")] public string ExamineGroup = "worn-stats";
+
+        [DataField("examinePriorityCoefficient")] public int ExaminePriorityCoefficient = 4;
+        [DataField("examinePriorityFlat")] public int ExaminePriorityFlat = 5;
```

Comment:
I think it'd be better to just have these as a flat number. Might also be best to add some constants in a static class somewhere so if something wants to say "Armor + 1" it's easy to do so.

Also if this is required on every examine group it should also be an interface, this also lets you inherit the xml doc too.

Reply:
I made it a single flat number and made it an interface, if thats what you meant?

Not sure where I should add a static class with constants